elliottlib/cli/find_bugs_kernel_cli.py

```diff
@@ -303,10 +287,10 @@ def _new_jira_fields_from_bug(bug: Bug, ocp_target_version: str, kmaint_tracker:
               is_flag=True,
               default=False,
               help="Update summary, description, etc for already cloned Jira bugs. Must be used with --clone")
-@click.option("--comment",
+@click.option("--update-tracker",
```

Comment:
We need to update aos-cd-jobs for the new option name.

Reply:
https://github.com/openshift-eng/aos-cd-jobs/pull/3882